stock_force_assign_by_type/models/stock_picking.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Tecnativa S.L. - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Picking(models.Model):
+    _inherit = "stock.picking"
+
+    picking_type_force_reservation = fields.Boolean(
+        related="picking_type_id.force_reservation"
+    )
+
+
+class PickingType(models.Model):
+    _inherit = "stock.picking.type"
```

Comment:
This is something that should go in its own file, as it's not a one2many model. And please name the classes with their complete name (`Picking` > `StockPicking`, `PickingType` > `StockPickingType`). If Odoo doesn't do this according guidelines doesn't mean that we shouldn't follow them.

Reply:
done! thanks